docs/developing/testing.rst

```diff
@@ -39,9 +33,24 @@ Here is an bare-bones example of the basic layout for a test:
       # Put your tests here
       self.assertEqual(1, 1)
 
+Every test should include comments that explain, almost in narrative form, what the test is doing and what the expected results are.
+
+Generally, there are two ways that you test Ion's code. These all not comprehensive, but should work for most cases. The first is calling methods directly; the second is making a GET or POST request to the view that you are interested in testing. The best tests utilize both. After you do either/both of these to call the code, you should use assertions to see if the code behaved as expected. A list of assertion options can be found `here <https://docs.python.org/3/library/unittest`_.html#assert-methods>`. A useful tool for making requests to the view is ``self.client``. More documentation on that can be found `here <https://docs.djangoproject.com/en/3.2/topics/testing/tools/>`_. Alternatively, just search through Ion's testing files for examples of using `self.client`.
+
+Good Testing Examples
+=====================
+
+Ion has lots of tests; the following are examples of very well written ones. Note the abundance of comments and the attention to detail. Test everything.
```

Comment:
```suggestion
Ion has lots of tests; the following are examples of very well-written ones. Note the abundance of comments and the attention to detail. Test everything.

- `Polls <https://github.com/tjcsl/ion/blob/dev/intranet/apps/polls/tests.py>`_ - Note the thoroughness of the checks; assertions are made even when nothing should have changed.
- `Emailfwd <https://github.com/tjcsl/ion/blob/dev/intranet/apps/emailfwd/tests.py>`_ - Note the use of `self.client` and checking `messages` generated by the page.
- `Eighth <https://github.com/tjcsl/ion/tree/dev/intranet/apps/eighth/tests>`_ - If a testing method has been used in Ion, it's in ``eighth``. This is the most complicated and also the most thorough set of tests that Ion has.
```
`If a testing method has been used in Ion` might need a little bit of rewording - it comes across as a touch confusing.